gitinstall-hp.md

```diff
@@ -0,0 +1,98 @@
+# Installing git on HP-UX
+
+Topics in this document:
+
+* [Installing via Depot Packages](#installing-via-depot-packages)
+* [Installing via Internet](#installing-via-internet)
+  * [Caching Depot Packages](#caching-depot-packages)
+* [Installing via Source Code](#installing-via-source-code)
+* [Basic git functionality test](#basic-git-functionality-test)
+
+## Installing via Depot packages
+
+This is the recommended install method if you have read access to the wsscfs network share.
+
+* Login as root on the target HPv3 system : `ssh root@myHPv3system`
+* Create a temporary folder to hold the depot files : `mkdir -p /root/GIT`
+* Copy over the depot packages to `/root/GIT`
+  * HPv3 ia64 : `\redmond\wsscfs\OSTCData\NixArchive\SourceCodeManagement\git\HP-UX\v3ia64`
+  * HPv3 parisc : `\redmond\wsscfs\OSTCData\NixArchive\SourceCodeManagement\git\HP-UX\v3risc`
+* Unzip the compressed depot files : `gunzip /root/GIT/*.gz`
+* Install the git dependencies then git itself. The dependencies were [obtained using depothelper](#caching-depot-packages).
+
+For HPv3 ia64:
```

Comment:
So if this is for ia64, what about parisc? Do you have instructions for those?

Was there some way this list was generated? If so, you should comment on how to generate this list.

Lastly, for Solaris, I generated command scripts that you can run. Should you do that here too, or just leave it as is?